### DISCUSSION TITLE
Hello marcel,

### DISCUSSION THREAD

**Discussion by AMinuteAgo** [marceldobehere/MaslOS#5] [General]
Minute on a different account here, i think the current kernel panic screen is pretty good but if that happened to me i would simply die. Instead of it overlapping the current screen, why not make it a less intimidating screen? Like windows but green or something like that.
![Screenshot 2023-01-17 12 35 40 PM](https://user-images.githubusercontent.com/114044206/212987729-e0ae6e2d-1fb5-45c7-992b-70f694ac7e0f.png)

**Comment by AMinuteAgo** [comment]
**beans**

**Comment by marceldobehere** [comment]
I might add a crash color var sometime. But other than that it'll probs stay like that for now
